### ISSUE TITLE
Elevated left ventricular end-diastolic diameter

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7594]
**Preferred term label:**
Elevated left ventricular end-diastolic diameter

**Synonyms**
Elevated left ventricular internal diameter in diastole
Elevated LVEDD
Dilated LVEDD

**Definition (free text, please give PubMed ID)**
The LV end-diastolic internal diameter was measured from two-dimensional (2D) images in the parasternal long-axis view, timed with mitral valve closure at the level of the mitral valve chordae.
PMID:25525505

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
612201 
PMID: 18614783  5/11

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
